packages/compass-global-writes/src/store/reducer.ts

```diff
@@ -433,6 +593,64 @@ export const createShardKey = (
   };
 };
 
+// Exporting this for test only to stub it and set
+// its value. This enables to test cancelSharding action.
+export const showConfirmation = showConfirmationModal;
```

Comment:
Just FYI, if you test the stores following the [redux testing guides](https://redux.js.org/usage/writing-tests) that suggest to test stores through connected UI, you don't need to come up with workarounds for your testing setup

Reply:
fair point, these could've been full integration tests with UI as well.

Reply:
at this point I would leave them this way unless you feel strongly against it